{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "17f309fd",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Data Processing and Bioinformatics Libraries\n",
    "\n",
    "import pysam  # For reading and manipulating SAM/BAM/VCF files, commonly used for storing large-scale genetic data.\n",
    "import skbio  # scikit-bio: For bioinformatics, including analysis and visualization of biological sequence data.\n",
    "import vcfpy  # For parsing and manipulating VCF (Variant Call Format) files that contain genetic variation data.\n",
    "import h5py  # For working with HDF5 file format, useful for storing large datasets efficiently.\n",
    "\n",
    "# Data Cleaning and Management Libraries\n",
    "import pandas as pd  # For handling structured data (e.g., CSV, Excel) and performing data cleaning, transformation, and analysis.\n",
    "import numpy as np   # For numerical operations on arrays and matrices, often used for large-scale data manipulation in bioinformatics.\n",
    "\n",
    "# Data Visualization Libraries\n",
    "import plotly        # For creating interactive, web-based visualizations, useful for exploring and presenting genetic data.\n",
    "import seaborn       # For creating statistical data visualizations, often used for producing more complex and informative plots than standard matplotlib."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2b7130d8",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
